diann/1.9.1/Dockerfile

```diff
@@ -0,0 +1,45 @@
+################## BASE IMAGE ######################
+FROM ubuntu:latest
+
+################## METADATA ######################
+LABEL base_image="ubuntu:latest"
+LABEL version="1"
+LABEL software="diann"
+LABEL software.version="1.9.1"
+LABEL about.summary="DIA-NN - a universal software for data-independent acquisition (DIA) proteomics data processing."
+LABEL about.home="https://github.com/vdemichev/DiaNN"
+LABEL about.documentation="https://github.com/vdemichev/DiaNN"
+LABEL about.license_file="https://github.com/vdemichev/DiaNN/LICENSE.txt"
+LABEL about.license="Proprietary - See LICENSE.txt for details"
```

Comment:
Should be 'Custom License' IIRC

Reply:
Thanks. Will change!

Reply:
@npinter We have to be sure that the container can be distributed. As far as I know, we can't distribute this version. It is better to contact the developer of DIANN before the release of this container.